File: SecondYear/jeezzzz_AjeeshRawal_2226cseai1021_2/Week_1/forLoop.java
import java.util.*;
public class Solution {


	public static void main(String[] args) {
		
		/* Your class should be named Solution.
	 	* Read input as specified in the question.
	 	* Print output as specified in the question.
		*/
        int f=0,s=1,t,n;
		Scanner sc=new Scanner(System.in);
		n=sc.nextInt();
		t=s+f;
		if(n==1)
		System.out.println("1");
		else
		{
		for(int i=2;i<n;i++)
		{
			f=s;
			s=t;
			t=s+f;
		}
		System.out.println(t);
		}



		
	}

}

File: SecondYear/aayushker_AayushkerSingh_2226CSE(AI)1010_2/Week_1/For_loop.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){
        /*
        *  Write your code here. 
         *  Read input as specified in the question.
         *  Print output as specified in the question.
        */
      int a=0,b=1,c=1,i,t;
      cin>>t;
      for(i=1;i<t;i++){
              c=a+b;
              a=b;
              b=c;
      }
      cout<<c;
      return 0;
}
